app/src/main/java/movierating/repositories/UserRepository.java

```diff
@@ -3,14 +3,16 @@
 import movierating.models.Movie;
 import movierating.models.User;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class UserRepository {
 
-    private final List<User> userList;
+    private List<User> userList;
```

Comment:
Why is this a List not a Map<String, User>?
Because I see the usage patterns are of the type get the User object by name.

Reply:
No need to separately mention userList or userMap, can be simply written as users. List or Map can be directly figured out by its type.